docs/source/guide_numpyvectorarray.md

```diff
@@ -0,0 +1,51 @@
+---
+jupytext:
+  text_representation:
+   format_name: myst
+jupyter:
+  jupytext:
+    cell_metadata_filter: -all
+    formats: ipynb,myst
+    main_language: python
+    text_representation:
+      format_name: myst
+      extension: .md
+      format_version: '1.3'
+      jupytext_version: 1.11.2
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
+```{code-cell}
+:load: myst_code_init.py
+:tags: [remove-cell]
+
+```
+
+# How to Convert NumPy Arrays to pyMOR VectorArrays?
+
+You may have a 2D NumPy array `X`,
+where rows correspond to high-dimensional snapshots,
+and you want to convert it to a {{VectorArray}}
+to be able to pass it to pyMOR's algorithms.
+
+As an example, let us use
+
+```{code-cell}
+import numpy as np
+X = np.ones((5, 100))
+X.shape
+```
+
+Then use the following to convert it to a pyMOR {{NumpyVectorArray}}:
+
+```{code-cell}
+from pymor.vectorarrays.numpy import NumpyVectorSpace
+V = NumpyVectorSpace.from_numpy(X)
+len(V), V.dim
+```
+
+In particular, note that {{VectorSpace}} subclasses
+(in this case, {{NumpyVectorSpace}})
+should be used to build {{VectorArrays}}.
```

Comment:
I think it would be helpful to also show how to convert back. Especially that you need to transpose.

Reply:
Before moving forward with this, I'd really like to further discuss if we want this. IMHO, there should be a 'working with VectorArrays' tutorial, which I intend to write at some point. If there is such an tutorial, do we want to duplicate the information here? The two concerns I have:

1. Nobody wants to write documentation. Maintaining existing documentation even less. The 'technichal overview' is a good example how documentation can rot. So the amount of text should be kept at a minimum.
2. I have seen several cases of documentation featuring different categories like (howto, getting started, examples, in-depth information, etc.), where I was really confused where to look to find a specific piece of information. As such, I would really like clearly defined entry points to the documentation. The separation between 'how to' and 'tutorial' seems to vague to me.